frontend/src/errors/AuthErrors.ts

```diff
@@ -0,0 +1,29 @@
+import { PresentableError } from "../types/ErrorTypes";
+
+export const authErrors: Record<string, PresentableError> = {
```

Comment:
do you think that instead of just a regular string, we could use enums instead? The same idea would apply to the errors on the backend

Reply:
ooh I like it